ts/src/bingx.ts

```diff
@@ -5156,7 +5293,7 @@ export default class bingx extends Exchange {
                 const startTimeReq = market['spot'] ? 'startTime' : 'startTs';
                 request[startTimeReq] = since;
             } else if (market['swap']) {
-                request['startTs'] = now - 7776000000; // 90 days
+                request['startTs'] = now - 30 * 24 * 60 * 60 * 1000; // 30 days for swap
```

Comment:
90 days was incorrect, it should be max 30 days: https://bingx-api.github.io/docs/#/en-us/swapV2/trade-api.html#Query%20historical%20transaction%20orders